### DISCUSSION TITLE
How can I deploy a cluster with 3 nodes and ensure that It will recover when a node down

### DISCUSSION THREAD

**Discussion by xtyDoge** [qdrant/qdrant#2609] [Q&A]
For example, I have 3 nodes, with hostname as:

- node1.local
- node2.local
- node3.local

Then how can I start nodes with --bootstrap and --uri params, and the sequence to start nodes?

**Comment by timvisee** [comment]
You start the first peer with `--uri`, to tell it what URI to expose on.

Then you start the other peers with `--bootstrap`, to tell what peer to bootstrap from.

You can find more on this here: https://qdrant.tech/documentation/guides/distributed_deployment/
